core/types/block.go

```diff
@@ -430,3 +430,12 @@ func (b *Block) Hash() common.Hash {
 }
 
 type Blocks []*Block
+
+// WriteStatus status of write
+type WriteStatus byte
```

Comment:
As mentioned elsewhere, i would keep this inside `blockchain` and avoid exposing it to the outside the package.